src/types/components.d.ts

```diff
@@ -318,6 +318,15 @@ export interface NewsTeaserItemProps {
 }
 export class NewsTeaserItem extends Component<NewsTeaserItemProps> {}
 
+export interface ProductListItemProps {
+  title: string;
+  description: string;
+  price?: number;
```

Comment:
You declared the price as required in your prop-decorator. Please synchronize the type with the property definition.